examples/powertools-examples-core/pom.xml

```diff
@@ -118,4 +122,72 @@
         </plugin>
       </plugins>
     </build>
+    <profiles>
+        <!-- Use a profile to enforce AspectJ version 1.9.7 if we are Java 1.8, otherwise we'll get class
+             version mismatch issues. All subsequent Java releases build with the default AspectJ configuration
+              on the project. -->
+        <profile>
+            <id>jdk8</id>
+            <activation>
+                <jdk>(,11)</jdk> <!-- 8 -->
+            </activation>
+            <properties>
+                <aspectj.version>1.9.7</aspectj.version>
+            </properties>
+            <dependencyManagement>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.aspectj</groupId>
+                        <artifactId>aspectjtools</artifactId>
+                        <version>${aspectj.version}</version>
+                    </dependency>
+                </dependencies>
+            </dependencyManagement>
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>dev.aspectj</groupId>
+                            <artifactId>aspectj-maven-plugin</artifactId>
+                            <version>${aspectj.plugin.version}</version>
+                            <configuration>
+                                <source>${maven.compiler.source}</source>
+                                <target>${maven.compiler.target}</target>
+                                <complianceLevel>${maven.compiler.target}</complianceLevel>
+                                <aspectLibraries>
+                                    <aspectLibrary>
```

Comment:
In each particular example, the set of `aspectLibraries` specified must be reflected in both the main pom and the profile override